H_ecto_models.md

```diff
@@ -444,21 +444,20 @@ Once a user submits the form rendered from `new.html` above, the form elements a
 def create(conn, %{"user" => user_params}) do
   changeset = User.changeset(%User{}, user_params)
 
-  if changeset.valid? do
-    Repo.insert!(changeset)
-
-    conn
-    |> put_flash(:info, "User created successfully.")
-    |> redirect(to: user_path(conn, :index))
-  else
-    render(conn, "new.html", changeset: changeset)
+  case Repo.insert(changeset) do
+    {:ok, _user} ->
```

Comment:
Can I just take a moment to say how much I love the new tuple-style return values? :heart: